### ISSUE TITLE
Increase size of direction arrow in POI search results

### ISSUE THREAD

**Issue by Fizzie41** [osmandapp/OsmAnd#17765]
Not sure if this should be a feature request, a bug report or something else, but here goes.

When you search for a POI category, you get a list of possible alternatives, sorted by distance.

As I've only just found out (!), there's also an arrow indicating what direction the POI is in.

Would it be possible to either make those arrows larger, or make it easier to see what direction they are actually pointing, possibly by making them dual colour e.g. a red tip on a black arrow? At the moment, it's just a small triangle which is hard to make out, especially for older eyes!
![Screenshot_2023-07-31-15-16-22-47_014aeb6b5e04ee7837dcdaecc5244511](https://github.com/osmandapp/OsmAnd/assets/35707521/1f24a4ce-f31b-4058-8217-7884936a7fc1)

**Comment by sonora** [comment]
Yes, true, the shape is unfortunate: It is almost an equilateral triangle, (with only one side being slightly dented). Anything which makes it clearer which way it points, e.g. streching it significantly to make it more like an arrow, would be better.

**Comment by tradavyvy** [comment]
Maybe making it a little bigger and use a V shape in stead of a filled in triangle.

**Comment by scaidermern** [comment]
Duplicate of #3032 (but good idea to raise this issue again).

**Comment by Fizzie41** [comment]
> Duplicate of #3032

Thanks! I did search before posting but didn't find that?

& just playing with simple possibilities, such as a sharp pointed V shape, the same size as the distance text font, or an arrow shape. Either would seem to help fix it?
![Direction arrows](https://github.com/osmandapp/OsmAnd/assets/35707521/b0969eab-d5dd-4b45-8fa8-8ce7aacebce6)
